CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 The following is a curated list of changes in the Enact sandstone module, newest changes on the top.
 
+## [unreleased]
+
+### Added
+
+- `sandstone/Alert` and `sandstone/Input` support for portrait mode
+- `sandstone/Icon` supported icon list, adding a new icon `wallpaper`
+- `sandstone/VideoPlayer` props `backButtonAriaLabel` and `onBack` to provide a way to exit video player via touch
+
+### Changed
+
+- `sandstone/Panels.Header` and `sandstone/RadioItem` to use `onClick` instead of `onTap` for touch support
```

Comment:
Why do you exclude Sample fix?
https://github.com/enactjs/sandstone/pull/793/files#diff-e8f6b68eaefaf44020bc3ff1fc7a18f1a73690ff91da72e19756be6bf9edef04

Reply:
Samples fix are added.